lib/yao/resources/action.rb

```diff
@@ -0,0 +1,16 @@
+module Yao::Resources
+  module Action
+    def action(id, query)
+      res = POST(action_path(id)) do |req|
+        req.body = query.to_json
+        req.headers['Content-Type'] = 'application/json'
```

Comment:
yes it is.
comment out this part
`req.headers['Content-Type'] = 'application/json'`
* request debug
```
#<Faraday::Env 
	@method=:post 
	@body="{\"os-stop\":null}" 
	@url=#<URI::HTTP http://example.api.com:8774/v2/b598bf98671c47e1b955f8c9660e3c44/servers/cee879b6-aefd-4a0d-8f61-85a5b0ecaec5/action> 
	@request=#<Faraday::RequestOptions (empty)> 
	@request_headers={"User-Agent"=>"Faraday v0.9.2", "Content-Type"=>"application/x-www-form-urlencoded", "X-Auth-Token"=>"3c28a51d13624c45827785c9a35ba7f3"} 
	@ssl=#<Faraday::SSLOptions (empty)>
>
```
json is not included in the request_headers.